### ISSUE TITLE
webutil.user_exists state uses deprecated htpasswd.useradd_all module

### ISSUE THREAD

**Issue by Talkless** [saltstack/salt#34241]
### Description of Issue/Question
When running webutil.user_exists state, salt minion issues warning:
```
2016-06-23 15:36:15,644 [py.warnings                                                 ][WARNING ][4810] /usr/lib/python2.7/dist-packages/salt/modules/htpasswd.py:69:     DeprecationWarning: 'htpasswd.useradd_all' has been deprecated in favor of 'htpasswd.useradd'. It's functionality will be removed in Salt Nitrogen. Please migrate     to using 'useradd' instead of 'useradd_all'.
```
Probably it's initiated here:
https://github.com/saltstack/salt/blob/v2016.3.1/salt/states/htpasswd.py#L71

Deprecation warning is here:
https://github.com/saltstack/salt/blob/v2016.3.1/salt/modules/htpasswd.py#L68

### Versions Report
Salt Version:
           Salt: 2016.3.1
 
Dependency Versions:
           cffi: Not Installed
       cherrypy: 3.2.2
       dateutil: 1.5
          gitdb: 0.5.4
      gitpython: 0.3.2 RC1
          ioflo: Not Installed
         Jinja2: 2.6
        libgit2: Not Installed
        libnacl: Not Installed
       M2Crypto: Not Installed
           Mako: 0.7.0
   msgpack-pure: Not Installed
 msgpack-python: 0.3.0
   mysql-python: 1.2.3
      pycparser: Not Installed
       pycrypto: 2.6.1
         pygit2: Not Installed
         Python: 2.7.3 (default, Jun 21 2016, 18:38:19)
   python-gnupg: Not Installed
         PyYAML: 3.10
          PyZMQ: 13.1.0
           RAET: Not Installed
          smmap: 0.8.2
        timelib: Not Installed
        Tornado: 4.2.1
            ZMQ: 3.2.3
 
System Versions:
           dist: debian 7.11 
        machine: x86_64
        release: 3.2.0-4-amd64
         system: Linux
        version: debian 7.11

**Comment by Ch3LL** [comment]
@Talkless nice catch. Looks like we need to change this to make sure come nitrogen release we don't run into an issue. Thanks

**Comment by rallytime** [comment]
@Talkless Please see the fix in #35788.

**Comment by Talkless** [comment]
Looks good, thanks!
